content/02.main-text.md

```diff
@@ -15,6 +15,9 @@ We encouraged anyone interested to contribute by proposing changes or additions.
 Although we invited some specific experts to participate, most authors discovered the manuscript organically through conferences or social media, deciding to contribute without solicitation.
 In total, the Deep Review attracted {{deep_review_authors}} authors, who were not determined in advance, from 20 different institutions.
 
+Papers with hundreds or thousands of authors ("hyperauthorship") are on the rise, such as the article announcing the discovery of the Higgs Boson [@doi:https://doi.org/10.1103/PhysRevLett.114.191803] (5000 authors) and the article reporting the Drosophila genome [@doi:https://doi.org/10.1534/g3.114.015966] (1000 authors).
```

Comment:
I wouldn't say it's common -- I had never heard of it -- but I did come across it while looking for these things. Specifically, it's mentioned here with the following quote: http://archive.sciencewatch.com/newsletter/2012/201207/multiauthor_papers/ 

> To borrow a term credited to Indiana University information scientist Blaise Cronin, “hyperauthorship” would seem to be flourishing—driven in particular, as we’ll see, by an international undertaking in high-energy physics that recently made world headlines.

- I removed the parenthetical because it doesn't really add anything.
- Fixed the citations.
- Changed wording for Drosophila but kept the slightly longer description of the Higgs Boson, since I amended that to remove the implication it was the announcement.